test-common-blackduck/src/main/java/com/synopsys/integration/alert/test/common/blackduck/BlackDuckResponseTestUtility.java

```diff
@@ -48,6 +49,9 @@ public ProjectVersionComponentVersionView createProjectVersionComponentVersionVi
         meta.setLinks(List.of(resourceLink));
         projectVersionComponentVersionView.setMeta(meta);
 
+        VersionBomOriginView versionBomOriginView = new VersionBomOriginView();
+        projectVersionComponentVersionView.setOrigins(List.of(versionBomOriginView));
```

Comment:
Could just do projectVersionComponentVersionView.setOrigins(List.of(new VersionBomOriginView())); if versionBomOriginView isnt used again

Reply:
This will be fixed in a followup PR containing test improvements for this section.